.github/pull_request_template.md

```diff
@@ -9,11 +9,29 @@
 ## Issue link
 <!-- Add a link to the issue -->
 
-## Checklist before requesting a review
-- [ ] I have performed a self-review of my code
-- [ ] I have added thorough, applicable tests that validate my changes
+## Pre-review checklist
+
+If some steps are not applicable or in your judgment not necessary, remove and explain why rather than leave checkmarks blank.
+
+- [ ] I've tested the code as an end user would and included appropriate testing steps 
+- [ ] For all new functionality, I have added thorough, applicable tests that validate my changes
+- [ ] For improvements to existing functionality, I've followed the [campsite rule](https://github.com/encodium/.github-private/blob/main/profile/pages/dev-standards.md#git), leaving it a little better than I found it. 
 - [ ] The added feature contains necessary observability
 - [ ] I have made corresponding changes to the documentation
 
+## Review checklist
+- [ ] Review with peer on team.
+- [ ] Review with team 'merger' or on-duty merger. Details [here](https://github.com/encodium/.github-private/blob/main/profile/pages/dev-standards.md#git). 
+
+## Peer review checklist
+
+- [ ] Make sure the pre-review checklist has been completed.
+- [ ] I've run the code as an end user would.
+
+## Merger review checklist
+
+- [ ] Make sure the pre-review checklist has been completed.
+- [ ] I've run the code as an end user would.
```

Comment:
> I've run the code as an end user would.

Sometimes it would be helpful to indicate how to reproduce or run as an end-user would run. Where do you think the best place to document those steps would be?

Reply:
Good call out. The pre-review checklist that the author fills out does mention documenting this, but we don't have a section in the template for it. We should add one.

Reply:
Wouldn't those steps be supplied in the `Testing information` section of the PR?

Reply:
Oh yeah, above the fold. We do totally have this field already.